### PR TITLE
raidboss: DSR doorboss WIP triggers and timeline

### PR DESCRIPTION
**Pull request by valarnin**
Extremely WIP, only up to doorboss headmarkers so far, and a wall of placeholder triggers for all the various abilities and statuses in the fight that I'm pruning as I work through the log file I have available.

Only tested in raidemulator so far.

Timeline is also incomplete, not cleaned up, and doesn't have the command at the top yet.

Feel free to steal parts of what I have here to supplement what you're working on, otherwise I'll update this as I make progress on triggers.

### CODE REVIEW

**Comment by quisquous** [comment]
Thanks! I cleaned up the timeline in #4344 and tested against the log file from discord.

File: ui/raidboss/data/06-ew/ultimate/dragonsongs_reprise_ultimate.ts [hunk 1]
```diff
@@ -29,7 +41,2194 @@ console.assert(getHeadmarkerId);
...
+      outputStrings: {
+        aoe: Outputs.aoe,
+      },
+    },
+    // 0x62D1 = Holy Shield Bash, single target damage on tethered player
+    // 0x63EB = Holy Shield Bash, line charge aoe at tethered player
+    // Charge applies 0xC5D Down for the Count
+    // Followup is 0x62D2 (single target) and 0x62D3 (aoe) Holy Bladedance tank busters
+    {
+      id: 'DSR Empty Dimension',
```
Comment (quisquous) [inline ui/raidboss/data/06-ew/ultimate/dragonsongs_reprise_ultimate.ts:91]: Empty = In + Tank Tether, Full = Out + Tank Tether.

Comment (valarnin) [reply ui/raidboss/data/06-ew/ultimate/dragonsongs_reprise_ultimate.ts:91]: Ah, was wondering what the Full one was, classic FF 50/50 it didn't even happen once in the instance I was in 🙃

Comment (quisquous) [reply ui/raidboss/data/06-ew/ultimate/dragonsongs_reprise_ultimate.ts:91]: Errr actually on second thought I wonder a little if the first one is always empty and the second always full (or random)? I haven't looked in too much detail.

Comment (valarnin) [reply ui/raidboss/data/06-ew/ultimate/dragonsongs_reprise_ultimate.ts:91]: Got Empty on both every single run. That's why I was confused lol.

Comment (quisquous) [reply ui/raidboss/data/06-ew/ultimate/dragonsongs_reprise_ultimate.ts:91]: Looking more closely, I think it is always Empty for (1) and then Empty/Full for (2).

Comment (Akurosia) [reply ui/raidboss/data/06-ew/ultimate/dragonsongs_reprise_ultimate.ts:91]: for my group first was alsways empty and second was random

**Comment by valarnin** [comment]
Closing this at this point, since I've had 0 time to work on triggers due to work.

**Pull request closed without merging**